test/cli/package-import-parent-package-conflict/parent/__package.rb

```diff
@@ -1,5 +0,0 @@
-# typed: strict
```

Comment:
(ditto the question about about why delete this test vs capture the change to the cli output)

Reply:
The output turned into `No errors! Great job.`, so it seemed not as valuable as it was before. I can add them back in and update the output instead.